### PR TITLE
Tests: Add info about slow tests.

### PR DESCRIPTION
**Pull request by lezama**
Now when running phpunit you should see the following enriched output:

```
You should really fix these slow tests (>150ms)...
 1. 17859ms to run WP_Test_Jetpack_PHP_Lint:test_php_lint
 2. 15209ms to run WP_Test_Jetpack_Heartbeat:test_cron_exec
 3. 15177ms to run WP_Test_Jetpack_Sync_Module_Stats:test_dont_send_expensive_data_on_heartbeat
 4. 15003ms to run WP_Test_Jetpack_Sync_Module_Stats:test_sends_stats_data_on_heartbeat
 5. 9366ms to run WP_Test_Jetpack_Sync_Themes:test_install_edit_delete_theme_sync
 6. 8812ms to run WP_Test_Jetpack_Sync_Plugins:test_installing_and_removing_plugin_is_synced
 7. 6037ms to run WP_Test_Jetpack_Sync_Sender:test_rate_limit_how_often_sync_runs_with_option
 8. 6023ms to run WP_Test_Jetpack_Sync_Sender:test_limits_execution_time_of_do_sync
 9. 6006ms to run WP_Test_Jetpack_Sync_Queue:test_queue_lag
 10. 4164ms to run WP_Test_Jetpack_Sync_Functions:test_force_sync_callabled_on_plugin_update
...and there are 68 more above your threshold hidden from view

Time: 3.44 minutes, Memory: 138.25MB
```

### CODE REVIEW

**Comment by enejb** [comment]
❤️

**Comment by enejb** [comment]
Another fix I think we can do is move slow tests such as phpunit --filter=WP_Test_Jetpack_PHP_Lint
so that it only runs once. Per config instead of on mu and single sites. This would help a lot

File: tests/php/bootstrap.php [hunk 1]
```diff
@@ -111,3 +111,5 @@ function in_running_uninstall_group() {
 	global  $argv;
 	return is_array( $argv ) && in_array( '--group=uninstall', $argv );
 }
+
```
Comment (enejb) [inline tests/php/bootstrap.php]: Lets add a comment here.

**Comment by lezama** [comment]
Thanks!

**Pull request merged**